lunatrace/bsl/license-worker/pkg/vulnerability/advisory/osv.go

```diff
@@ -13,6 +13,7 @@ package advisory
 
 import (
 	"encoding/json"
+	"github.com/facebookincubator/nvdtools/cvss3"
```

Comment:
This is doing more than just license-worker stuff now, this is a general internal CLI tool.  I think we should consider renaming this thing or moving this code somewhere else.  Id go for renaming this to lunatrace-internal-cli probably

Reply:
at least like..package worker